qiita_pet/handlers/api_proxy/user.py

```diff
@@ -0,0 +1,53 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) 2014--, The Qiita Development Team.
+#
+# Distributed under the terms of the BSD 3-clause License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# -----------------------------------------------------------------------------
+from __future__ import division
+
+from qiita_core.util import execute_as_transaction
+
+
+@execute_as_transaction
+def user_jobs_get_req(user):
+    """Gets the json of jobs
+
+    Parameters
+    ----------
+    prep_id : int
+        PrepTemplate id to get info for
+    user_id : str
+        User requesting the sample template info
+
+    Returns
+    -------
+    dict of objects
+    {'status': status,
+     'message': message,
+     'template': {sample: {column: value, ...}, ...}
+    """
+
+    response = []
+    cmds = {}
+    for j in user.jobs():
+        cmd = j.command
+        if cmd not in cmds:
+            cmds[cmd] = cmd
+        ccmd = cmds[cmd]
```

Comment:
Not necessary but I like it cause that makes the subsequent calls smaller. However, we are only retrieving the name of the command so change it a little bit.